packages/ui/useScroll/useScroll.js

```diff
@@ -812,6 +813,7 @@ const useScrollBase = (props) => {
 				scrollByPage(ev.keyCode);
 			}
 		} else {
+			props.preventScroll(ev);
```

Comment:
Let's add a guard to check if props.preventScroll exists or not.

Reply:
It's fixed.